docs/run.js

```diff
@@ -34,12 +34,12 @@ const scalingFactor = 100;
  */
 function run(map, firebaseAuthPromise, disasterMetadataPromise) {
   setMapToDrawLayersOn(map);
-  createToggles(map);
   snapAndDamagePromise =
       convertEeObjectToPromise(ee.FeatureCollection(snapAndDamageAsset));
-  createAndDisplayJoinedData(
-      map, initialPovertyThreshold, initialDamageThreshold,
-      initialPovertyWeight);
+  const initialTogglesValuesPromise =
+      setUpInitialToggleValues(disasterMetadataPromise);
+  initialTogglesValuesPromise.then(() => createToggles(map));
```

Comment:
why not have setUpInitialToggleValues do everything instead of passing things around like this?

Reply:
Done